SoftwareGuide/Cover/Source/VWHistogramHSV.cxx

```diff
@@ -66,19 +66,19 @@ int main(int argc, char * argv[] )
     return -1;
   }
 
-  typedef unsigned char                        PixelComponentType;
-  typedef itk::RGBPixel<PixelComponentType>    InputPixelType;
-  typedef unsigned short                       OutputPixelType;
+  using PixelComponentType =  unsigned char                        ;
```

Comment:
May be too nitpick, but I would **eliminate the white spaces before the semicolon**.
I cannot remember whether this is allowed by the `KWStyle` checker.
There are multiple instances.
Thanks.